test/integration/broker.test.js

```diff
@@ -36,27 +37,27 @@ describe('broker: end-to-end', () => {
             port: trackerPort,
             id: 'tracker'
         })
-        broker1 = await startBroker({
-            name: 'broker1',
-            privateKey: broker1Key,
+        storageNode1 = await startBroker({
+            name: 'storageNode1',
+            privateKey: storageNodeAccount1.privateKey,
             networkPort: networkPort1,
             trackerPort,
             httpPort: httpPort1,
             wsPort: wsPort1,
             enableCassandra: true
         })
-        broker2 = await startBroker({
-            name: 'broker2',
-            privateKey: broker2Key,
+        storageNode2 = await startBroker({
```

Comment:
Does this test pass if we make 2/3 of the nodes non-storage? I think that has been the original intention...

Reply:
@harbu implements this refactoring in a separate PR